jablib/src/main/java/org/jabref/logic/importer/fetcher/citation/semanticscholar/SemanticScholarCitationFetcher.java

```diff
@@ -33,6 +34,12 @@ public String getAPIUrl(String entry_point, BibEntry entry) {
                 + "&limit=1000";
     }
 
+    public String getUrlForCitationCount(BibEntry entry) {
+        return SEMANTIC_SCHOLAR_API + "paper/" + "DOI:" + entry.getDOI().orElseThrow().asString()
+                + "?fields=" + "citationCount"
+                + "&limit=1000";
```

Comment:
I think, can pass `10` or even `1` as there will be only 0 or 1 thing returned.

Can you test if it works without this parameter?